src/crimson/os/seastore/cache.h

```diff
@@ -433,6 +446,7 @@ class Cache {
     LOG_PREFIX(Cache::get_extent);
     auto result = t.get_extent(offset, &ret);
     if (result != Transaction::get_extent_ret::ABSENT) {
+      ceph_assert(ret->is_fully_loaded());
```

Comment:
Same here.

IIUC the not-fully-loaded extents can be in both Cache (as CLEAN) and Transaction (as CLEAN_EXISTING), right?

Reply:
same above, it seems not-fully-loaded extent will not run into here?